src/Context/GetoptParser.php

```diff
@@ -265,12 +270,14 @@ protected function splitLongOptionInput($input)
      * @return bool
      *
      */
-    protected function longOptionRequiresValue($option, $value, $name)
+    protected function longOptionRequiresValue($option, $value, $name=null)
     {
         if ($option->param == 'required' && trim($value) === '') {
-            $this->errors[] = new Exception\OptionParamRequired(
-                "The option '$name' requires a parameter."
-            );
+            if ($name) {
```

Comment:
I added the strict comparison as the condition is not made, here, to check if the $name has a value, but to check if an error should be thown.